assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+  },
+  "language": [
+    "en"
+  ],
+  "maintainer": "",
+  "icon": "./assets/themes/campersites/caravan.svg",
+  "version": "0",
+  "startLat": 43.14,
+  "startLon": 3.14,
+  "startZoom": 14,
+  "widenFactor": 0.05,
+  "socialImage": "./assets/themes/campersites/Bar%C3%9Fel_Wohnmobilstellplatz.jpg",
+  "layers": [
+    {
+      "id": "caravansites",
+      "name": {
+        "en": "Camper sites"
+      },
+      "minzoom": 12,
+      "overpassTags": {
+        "and": [
+          "tourism=caravan_site"
+        ]
+      },
+      "title": {
+        "render": {
+          "en": "Camper site {name}"
+        },
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "Free camper site {name}"
+            }
+          }
+        ]
+      },
+      "description": {
+        "en": "camper sites"
+      },
+      "tagRenderings": [
+        "images",
+        {
+          "render": {
+            "en": "power supply"
+          },
+          "question": {
+            "en": "Does this place have a power supply?"
+          },
+          "freeform": {
+            "key": "power_supply",
+            "addExtraTags": [
+              "power_supply=yes"
+            ]
+          },
+          "condition": {
+            "and": []
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "power_supply=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has a power supply"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "power_supply=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have power supply"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "This place charges {charge}"
+          },
+          "question": {
+            "en": "How much does this place charge?"
+          },
+          "mappings": [],
+          "freeform": {
+            "key": "charge"
+          },
+          "condition": {
+            "and": [
+              "fee=yes"
+            ]
+          }
+        },
+        {
+          "render": {
+            "en": "This place is called {name}"
+          },
+          "question": {
+            "en": "What is this place called?"
+          },
+          "freeform": {
+            "key": "name"
+          },
+          "condition": {
+            "and": [
+              "name="
+            ]
+          }
+        },
+        {
+          "render": {
+            "en": "Sanitary dump station"
+          },
+          "question": {
+            "en": "Does this place have a sanitary dump station?"
+          },
+          "freeform": {
+            "key": "",
```

Comment:
Why define an empty key for freeform? This will cause bugs! Just remove the freeform block alltogether.

Reply:
done